3.9/data-science.md

```diff
@@ -0,0 +1,140 @@
+---
+Data Science
+==================
+
+ArangoDB's Graph Analytics and GraphML capabilities provide various solutions
+in data science and data analytics. Multiple data science personas within the
+engineering space can make use of ArangoDB's set of tools and technologies that
+enable analytics and machine learning on graph data. 
+
+ArangoDB, as the foundation for GraphML, comes with the following key features:
+
+- **Scalable**: Designed to support true scalability with high performance for
+ Enterprise use cases
+- **Simple Ingestion**: Easy integration in existing data infrastructure with
+ connectors to all leading data processing and data ecosystems
+- **Open Source**: Extensibility and Community
+- **NLP Support**: Built-In Text Processing, Search, and Similarity Ranking
+
+![ArangoDB Machine Learning Architecture](images/machine-learning-architecture.png)
+
+## Graph Analytics vs. GraphML
+
+This section classifies the complexity of the queries we can answer to - 
+like running a simple query that shows what is the path that goes from one node
+to another, or more complex tasks like node classification,
+link prediction, and graph classification.
+
+### Graph Query
+
+When running a query with AQL on a graph, the query goes from a vertex to an edge,
+and then the edge indicates what the next connected vertex will be.
+
+Graph queries can answer questions like _**Who can introduce me to person X**_?
+
+![Graph Query](images/graph-query.png)
+
+### Graph Analytics
+
+Graph analytics or graph algorithms is what you run on a graph if you want to 
+know aggregate information about your graph, while analyzing the entire graph.
+
+Graph analytics can answer questions like _**Who are the most connected persons**_?
+
+![Graph Analytics](images/graph-analytics.png)
+
+### GraphML
+
+When applying machine learning on a graph, you can predict connections, get 
+better product recommendations, and also classify vertices, edges, and graphs.
+
+GraphML can answer questions like 
+- _**Is there a connection between person X and person Y?**_
+- _**Will a customer churn?**_ 
+- _**Is this particular transaction Anomalous?**_
+
+![Graph ML](images/graph-ml.png)
+
+## Use Cases
+
+This section contains an overview of different use cases where Graph Analytics
+and GraphML can be applied.
+
+### Graph Analytics
+
+Graph Analytics consists of an analysis applied on graph-based data and has
+applicability in various fields such as marketing, fraud, supply chain,
+product recommendations, drug development, law enforcement, and cybersecurity.
+
+Graph Analytics uses an unsupervised
+learning method based on algorithms that are performing analytical processing
+directly on graphs stored in ArangoDB. This 
+[Distributed Iterative Graph Processing (Pregel)](graphs-pregel.html)
+is intended to help you gain analytical insights on
+your data, without having to use external processing systems.
+
+ArangoDB includes the following graph algorithms:
```

Comment:
Shall we link to the general Pregel page?

Reply:
There's already a link to the general Pregel page in the previous paragraph.